includes/Export.php

```diff
@@ -300,44 +370,115 @@ protected function dumpFrom( $cond = '' ) {
 				$prev = $this->db->bufferResults( false );
 			}
 
-			Hooks::run( 'ModifyExportQuery',
+			$result = null; // Assuring $result is not undefined, if exception occurs early
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)